doc/developers/contributing.rst

```diff
@@ -196,6 +196,25 @@ Contributing code
   so far. In order to claim an issue for yourself, please comment exactly
   ``take`` on it for the CI to automatically assign the issue to you.
 
+Video resources:
+----------------
+These videos are step-by-step introductions on how to contribute to
+scikit-learn, and are a great companion to the following text guidelines.
+Please make sure to still check our guidelines below, since they describe our
+latest up-to-date workflow.
+
+- Crash Course in Contributing to Scikit-Learn & Open Source Projects:
+
+    -`Video <https://youtu.be/5OL8XoMMOfA>`__
+    
+    -`Transcript <https://github.com/data-umbrella/event-transcripts/blob/main/2020/05-andreas-mueller-contributing.md>`__
```

Comment:
Having these as list places a bunch of white space to the right:

<img width="934" alt="Screen Shot 2020-10-08 at 1 58 39 PM" src="https://user-images.githubusercontent.com/5402633/95496204-7772d900-096e-11eb-85d3-9a308301e86f.png">
I think we can place the links on the right the of title to convey the same information.

```suggestion
- Crash Course in Contributing to Scikit-Learn & Open Source Projects:
  (`Video <https://youtu.be/5OL8XoMMOfA>`__,
   `Transcript <https://github.com/data-umbrella/event-transcripts/blob/main/2020/05-andreas-mueller-contributing.md>`__)
```